docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-creating-jira-issues-for-migration-wave_{context}"]
+= Creating Jira issues for a migration wave
+
+You can use a migration wave for creating Jira issues automatically for each application assigned to it.
+
+.Procedure
+
+. In *Migration* view, click *Migration waves*.
+. From the *Options* menu (three dots) of the migration wave for which you want to create Jira issues, select *Export to Issue Manager*.
++
+The *Export to Issue Manager* window opens.
+. Select the instance type (Jira Cloud or Jira Server/Datacenter).
+. Select the instance, project and issue type from the drop-down lists.
+. Click *Export*.
++
+The status of the migration wave on the *Migration waves* page changes to `Issues Created`. To see the status of each individual application of a migration wave, click the *Status* column.
++
+A separate Jira issue has been created for each application associated with the migration wave. The following fields of each issue are filled in automatically:
+
+* Title: `Migrate <app name>`
+* Reporter: Username of the token owner
+* Description: `Created by Konveyor`
+
+[NOTE]
+====
+Once the Jira issues have been created after you exported a migration wave to the Issue Manager, you can no longer change them from within the {ProductShortName} {WebName}. Even if you delete the migration wave, the Jira issues remain.
+====
+
```

Comment:
In the first sentence, is "after you exported a migration wave to the Issue Manager" relevant? That is, are you saying the same thing as "Once the Jira issues have been created,..." ? If the phrase is not relevant, I suggest deleting it. If it is relevant, I suggest starting the sentence with it "After you export a migration wave to the Issue Manager, you can no longer change any Jira issues related to the migration wave from inside the {ProductShortName} {WebName}."

The phrasing of the note leaves open the possibility that you can change the Jira issues from outside the UI. If that is so, consider specifying how to. If it's not so, consider de;etig "from within..."